Playground/CMakeLists.txt

```diff
@@ -1,20 +1,17 @@
 set(exe_name Playground)
-########################################
+###############################################
 # External Libraries
 xpGetExtern(externIncs externLibs PUBLIC boost)
-########################################
+###############################################
 set(${exe_name}_srcs
   main.cpp
   )
-
 source_group("" FILES ${${exe_name}_srcs})
-
-
 add_executable(${exe_name} ${${exe_name}_srcs})
+###############################################
 MESSAGE(STATUS "boost: " ${externLibs})
 MESSAGE(STATUS "boost: " ${externIncs})
 target_link_libraries(${exe_name} PUBLIC ${externLibs})
```

Comment:
asio and beast are header only, so you don't need to link anything.

Reply:
Oh, cool. Thanks